src/components/sbb-journey-summary/sbb-journey-summary.custom.d.ts

```diff
@@ -0,0 +1,28 @@
+declare type Leg = {
+  duration: number;
+};
+
+declare type TimeQuayWrapper = {
+  delay?: number;
+  /** True if platform change (de:Gleis-/Kante-/Steg-Änderung) */
+  quayChanged?: boolean;
+  /** A Quay (or platform or track) for any means of transport-mode / VehicleMode (train, bus, boat, etc.). */
+  quayName?: string;
+  /** planned arrival/departure time */
+  time: string;
+};
+
+declare type SummaryConfig = {
```

Comment:
@k-luise @kyubisation since this is a subtree of the same type used in timetable-row, any chance to store it only once and reference it ?